ext/pcre/tests/preg_replace_callback_array.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+preg_replace_callback_array()
+--FILE--
+<?php
+
+function f() {
+	throw new Exception();
+}
+
+try {
+var_dump(preg_replace_callback_array(array('/\w/' => 'f'), 'z'));
+} catch(Exception $e) {}
+
+function g($x) {
+	return "'$x[0]'";
+}
+
+var_dump(preg_replace_callback_array(array('@\b\w{1,2}\b@' => 'g'), array('a b3 bcd', 'v' => 'aksfjk', 12 => 'aa bb')));
+
+var_dump(preg_replace_callback_array(array('~\A.~' => 'g'), array(array('xyz'))));
+
+var_dump(preg_replace_callback_array(array('~\A.~' => create_function('$m', 'return strtolower($m[0]);')), 'ABC'));
```

Comment:
Is there a particular reason for using `create_function()` here instead of a regular anonymous function?